### ISSUE TITLE
Video subtitles?

### ISSUE THREAD

**Issue by Andronian42** [Andronian42/spdrFramePoster#4]
Implementing this kind of feature would be grueling, but interesting. Do it.

**Comment by Andronian42** [comment]
At some point this should be done through alt text.
I'll need to get a copy of the subtitles from the DVD release (that should work, right?)

**Comment by Andronian42** [comment]
With support for multiple platforms being necessary now, this will need to be rethought on a per-platform basis
